cuba.yml

```diff
@@ -405,3 +405,23 @@ CUBA_KEYS:
     definition: Work of adhesion per unit contact area
     shape: [1]
     type: double
+
+  MAJOR:
+    definition: Major attribute of an entity.
+    shape: [8]
+    type: string
+
+  MINOR:
+    definition: Minor version of a software tool with semantic versioning.
```

Comment:
to keep it general, as the cuba.yml defines "atomic values" of generic nature, how about changing definition to:

"Minor attribute of an entity"

then when we attach it to a VERSION cuds component, it becomes the minor version, one can there change the definition if needed to be more specific.